src/TodoItem.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const TodoItem = (props) => {
+    if(props.isCompleted){
+        console.log(props)
```

Comment:
To avoid errors in a large project and to make the code more readable for other programmers, do the following convention:
const TodoItem = ( { content, incomplete } ) ....
in that way the next programmer can know what the function needs

Reply:
Got it!
Like you can see sometimes I use the convention you mentioned and sometimes I used the props object. next time I'll try to identify the right parameters that the function is actually using and request only them.